src/titiler/core/titiler/core/algorithm/dem.py

```diff
@@ -224,9 +236,13 @@ def _range_check(datarange):
         if _range_check(datarange):
             raise ValueError(f"Data of {datarange} larger than 256 ** 3")
 
-        r = ((((data // 256) // 256) / 256) - (((data // 256) // 256) // 256)) * 256
-        g = (((data // 256) / 256) - ((data // 256) // 256)) * 256
-        b = ((data / 256) - (data // 256)) * 256
+        if self.use_nodata_height: 
+            data[img.array.mask[0]] = (self.nodata_height - self.baseval) / self.interval
```

Comment:
I don't think we need the 2 options, just `nodata_height` would be enough

Reply:
Thanks for the Optional, exactly what I was looking for!